SlevomatCodingStandard/Sniffs/Classes/RequireSingleLineMethodSignatureSniff.php

```diff
@@ -38,6 +44,13 @@ public function process(File $phpcsFile, $methodPointer): void
 		$signature = $this->getSignature($phpcsFile, $signatureStartPointer, $signatureEndPointer);
 		$signatureWithoutTabIndetation = $this->getSignatureWithoutTabs($signature);
 
+		if (count($this->includedMethodPatterns) !== 0) {
+			$isMethodSignatureInPatterns = $this->isMethodSignatureInPatterns($signatureWithoutTabIndetation, $this->includedMethodPatterns);
```

Comment:
Use `FunctionHelper::getName()` and check only method names with patterns.

And please remove useless variable `$isMethodSignatureInPatterns` and join both `if` together.